pkg/tagger/collectors/ecs_extract.go

```diff
@@ -0,0 +1,37 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+// +build docker
+
+package collectors
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/tagger/utils"
+	"github.com/DataDog/datadog-agent/pkg/util/docker"
+	ecsutil "github.com/DataDog/datadog-agent/pkg/util/ecs"
+)
+
+func (c *ECSCollector) parseTasks(tasks_list ecsutil.TasksV1Response) ([]*TagInfo, error) {
+	var output []*TagInfo
+	for _, task := range tasks_list.Tasks {
+		for _, container := range task.Containers {
+			tags := utils.NewTagList()
+			tags.AddHigh("ecs_arn", task.Arn)
```

Comment:
Is the arn useful as a tag? I have a hard time finding a use case for it.

Reply:
It uniquely identifies a resource, it can be useful if you have the same family name, version and task in different regions (or another AWS integration specific tag). Some AWS specific tags aren't being collected by the agent, so it could help. It is very specific I agree, I mostly integrated it per a contribution: https://github.com/DataDog/dd-agent/pull/3530

I'll remove it as a tag.